service/src/main/java/com/bankbazaar/kafka/service/producer/KafkaProducer.java

```diff
@@ -1,25 +1,41 @@
 package com.bankbazaar.kafka.service.producer;
 
 import com.bankbazaar.kafka.core.model.Data;
+import com.bankbazaar.kafka.dto.model.FileStatusDto;
 import com.bankbazaar.kafka.service.mapper.DataMapper;
 import com.bankbazaar.kafka.dto.model.DataDto;
-import lombok.NoArgsConstructor;
+import com.bankbazaar.kafka.service.model.Response;
+import com.bankbazaar.kafka.service.service.FileStatusService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.kafka.core.KafkaTemplate;
 import org.springframework.stereotype.Service;
 
+import java.util.Date;
+
 @Service
 public class KafkaProducer {
-    private  static  final  String TOPIC = "create_CSV";
+    private  static  final  String TOPIC = "Create_CSV";
 
     @Autowired
     private  KafkaTemplate<String, Data> kafkaTemplate;
+    @Autowired
+    private FileStatusService fileStatusService;
 
     @Autowired
     private DataMapper mapper;
 
-    public void sendData(DataDto data)
+    public Response sendData(DataDto data)
     {
+        FileStatusDto fileData = new FileStatusDto();
```

Comment:
Dtos are used in a scenario where we want to send data out of the application to a client. Its not required when we are calling other methods in the same service

Reply:
FileStatusDTo removed